nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/imports/converters/DifferentialFunctionClassHolder.java

```diff
@@ -221,7 +226,7 @@ private DifferentialFunctionClassHolder() {
 
                 }
 
-                fieldsForFunction.put(df.opName(), fieldNames);
+                fieldsForFunction.put(df.getClass().getName(), fieldNames);
```

Comment:
This looks like it could be quite a big change. Why go away from opName and use the Classname instead?